### PR TITLE
Cleanup subpackages and installable environments

### PR DESCRIPTION
**Pull request by mopsfelder**

### CODE REVIEW

File: lib/metapackage.py [hunk 1]
```diff
@@ -108,32 +108,6 @@ def update_metapackage(
         metapackage_yaml_file_path, yaml_install_dependencies_string,
         YAML_START_DELIMITER)
 
-    LOG.info("Updating release package spec file")
```
Comment (farosas) [inline lib/metapackage.py]: There is some code in here which relates to the update of the metapackage with information from the other packages, so I'd leave this message here.

File: lib/metapackage.py [hunk 2]
```diff
@@ -108,32 +108,6 @@ def update_metapackage(
...
-    with open(version_file_path) as version_file:
-        version_parts = version_file.readlines()[-1].strip().split("-")
-    new_metapackage_version = version_parts.pop(0)
-    if version_parts:
-        metapackage_milestone = version_parts.pop(0)
-    else:
-        metapackage_milestone = "%nil"
-
-    metapackage = rpm_package.RPM_Package.get_instance(
-        metapackage_name, distro)
```
Comment (farosas) [inline lib/metapackage.py]: We need this code.

File: lib/metapackage.py (hunk 1 above)
Comment (mopsfelder) [reply lib/metapackage.py]: Done

File: lib/metapackage.py [hunk 3]
```diff
@@ -109,31 +109,10 @@ def update_metapackage(
         YAML_START_DELIMITER)
```
Comment (farosas) [inline lib/metapackage.py:108]: ^ unused import up at line 5

**Review by farosas — APPROVED** [review]
LGTM, just please remember to make it pass validation

**Pull request merged**